### DISCUSSION TITLE
Port settings are not working properly.

### DISCUSSION THREAD

**Discussion by usuall** [zauberzeug/nicegui#1933] [Q&A]
### Question

I set the port to 8070. However, it starts with 8000.

#pip list
nicegui==1.4.1

#dev Env 
Windows 11 pro

ui.run(title='nicegui thanks', reload=False, native=True, port=8070, window_size=(1300, 500))

NiceGUI ready to go on http://localhost:8000        <===============
0.02s - Debugger warning: It seems that frozen modules are being used, which may
0.00s - make the debugger miss breakpoints. Please pass -Xfrozen_modules=off
0.00s - to python to disable frozen modules.
0.00s - Note: Debugging will proceed. Set PYDEVD_DISABLE_FILE_VALIDATION=1 to disable this validation.

**Comment by rodja** [comment]
This looks like a duplication of https://github.com/zauberzeug/nicegui/discussions/1457.
